soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -165,7 +166,6 @@ std::vector<uint32_t> buttonMap = {
     BTN_DDOWN,
     BTN_DLEFT,
     BTN_DRIGHT,
-    0
 };
```

Comment:
You removed the last entry in the enum but BTN_DRIGHT still has a comma after it.

Reply:
Woops, thanks